fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -650,6 +655,7 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType, Map<
                     parameterValue.setPrefix(prefix);
                 }
 
+                // DONT USE DateTime here
```

Comment:
yeah - my current WIP should address this.

Reply:
cool.  forgot i had added that comment there (probably when we were on webmeeting discussing it).  will leave in for now and you can remove it in yours